scheduler/src/test/scala/com/sky/kms/e2e/SchedulerResiliencySpec.scala

```diff
@@ -95,7 +97,7 @@ class SchedulerResiliencySpec extends SpecBase {
     def sourceWith(schedules: Seq[ScheduleEvent]): Source[ScheduleReader.In, (Future[Done], Future[Control])] = {
       val scheduleIds = List.fill(schedules.size)(UUID.randomUUID().toString)
 
-      val elements = (scheduleIds, schedules.map(_.some)).zipped.toIterator.map(_.asRight[ApplicationError]).toList
+      val elements = scheduleIds.lazyZip(schedules.map(_.some)).iterator.map(_.asRight[ApplicationError]).toList
```

Comment:
```suggestion
      val elements = scheduleIds.zip(schedules.map(_.some)).map(_.asRight[ApplicationError])
```

Don't think it needs to be lazy with 20 elements max being passed in.